projects/Mallard/src/helpers/files.ts

```diff
@@ -294,8 +294,10 @@ export const clearOldIssues = async (): Promise<void> => {
         issue =>
             !lastSevenDays()
                 .map(withPathPrefix(defaultSettings.contentPrefix))
-                .includes(issue) && issue !== 'issues',
+                .includes(issue) &&
+            issue !== `${defaultSettings.contentPrefix}/issues`,
```

Comment:
Can we have a test for the path so this doesn't happen again?

Reply:
Test added